### ISSUE TITLE
[$15][MSFT-204] [Registration] - Join Button and Show Button is not highlighted after hitting the tab button

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2868]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Hit Tab key until we reach to Join button below.
3. Observe the Join and Show Button. 

**Expected Result**
After entering all the fields the Show button and the Join button should be highlighted.

**Actual Result**
The Show Button and the Join button is not getting highlighted after hitting the tab button.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.3 Focus Order
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20JoinShow%20Highlighted

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096252 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096252 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096252 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@codeMinter PR https://github.com/appirio-tech/accounts-app/pull/248

**Comment by mishacucicea** [comment]
@PrakashDurlabhji Please look at the PR comments.

**Comment by PrakashDurlabhji** [comment]
@mishacucicea PR updated

@veshu @tosha5252

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096798 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096798 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @mishacucicea PR updated. not able to change the label else it is completed. 
https://github.com/appirio-tech/accounts-app/pull/248

**Comment by mishacucicea** [comment]
@PrakashDurlabhji Are you posting an update?

**Comment by PrakashDurlabhji** [comment]
@mishacucicea yes just some time

**Comment by PrakashDurlabhji** [comment]
@mishacucicea PR updated

**Comment by Oanh-and-only-Oanh** [comment]
@PrakashDurlabhji, what's the status on this?

**Comment by PrakashDurlabhji** [comment]
@Oanh-and-only-Oanh just 5 mins, and I hope highlight here means focused. correct?

because I am not able to acess following link it is not working.
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20JoinShow%20Highlighted

**Comment by PrakashDurlabhji** [comment]
@mishacucicea here is verification screenshot for join button. is this expected?
![2868](https://user-images.githubusercontent.com/28830240/62078144-8558a480-b269-11e9-8f79-af471357bc60.png)

**Comment by PrakashDurlabhji** [comment]
@mishacucicea PR updated https://github.com/appirio-tech/accounts-app/pull/248

**Comment by r0hit-gupta** [comment]
@PrakashDurlabhji please resolve merge conflicts. Thanks!

**Comment by PrakashDurlabhji** [comment]
@mishacucicea @r0hit-gupta  done

**Comment by nithyaasworld** [comment]
@nkumar-topcoder Its failing. I think this it not yet deployed. Please confirm. Thanks.

**Comment by nithyaasworld** [comment]
Its passing in https://accounts.topcoder-dev.com/member/registration 

Note: No highlight is done when tabbing over "Terms" and "Privacy Policy" links. As its not stated in the bug description, this will not be taken care as part of this ticket.

**Comment by nkumar-topcoder** [comment]
@mishacucicea #2868, #2775 and #2774 all fix overlap with similar files.

**Comment by PrakashDurlabhji** [comment]
Kindly note anyone who wishes to assign this  to himself that discussion for #2868 and #2774  is going on at #2775  and will be resolved soon.

**Comment by lakshmiathreya** [comment]
Verified on MacOS/Chrome & Win10/Chrome - Join and Show are now getting highlighted as expected.
<img width="1635" alt="Screenshot 2019-08-05 at 12 06 43 PM" src="https://user-images.githubusercontent.com/51733042/62443863-975ea980-b779-11e9-8567-509d25955144.png">

**Comment by mishacucicea** [comment]
@nkumar-topcoder I assume you want this retested due to a new merge, right?
cc @lakshmiathreya

**Comment by nkumar-topcoder** [comment]
> @nkumar-topcoder I assume you want this retested due to a new merge, right?
> cc @lakshmiathreya

Yes, i merged your latest dev branch fixes which had 2774. And wanted to re-test. Thanks

**Comment by PrakashDurlabhji** [comment]
@mishacucicea this issue was already solved by my PR you can verify for this at appirio-tech/accounts-app#248 which has code for highlight button by adding focus as stated in original issue

just #2775 was remaining and solved by rohit PR.

also #2774 was solved by my PR itself.

just all 3 issues were to be tested together hence was waiting on #2868 and #2774 till #2775 completed as per feedback.
and remaining part of #2775 was completed by rohit, so #2868 and #2774 has no much relation with it.

**Comment by tosha5252** [comment]
This is tested and working

Windows 10 env
Browser: Chrome

**Comment by tosha5252** [comment]
Tested and working

Windows 10 env
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096798<br/><br/>```This is an automated message for crazyk via Topcoder X```
